cmake/SeracMacros.cmake

```diff
@@ -148,13 +151,21 @@ macro(serac_add_tests)
 
     foreach(filename ${arg_SOURCES})
         get_filename_component(test_name ${filename} NAME_WE)
+        if (DEFINED arg_USE_CUDA)
+            set(test_name "${test_name}_cuda")
+        endif()
 
         blt_add_executable(NAME        ${test_name}
                            SOURCES     ${filename}
                            OUTPUT_DIR  ${TEST_OUTPUT_DIRECTORY}
                            DEPENDS_ON  ${arg_DEPENDS_ON}
                            FOLDER      serac/tests )
 
+        if (DEFINED arg_USE_CUDA)
+            target_compile_definitions(${test_name} PUBLIC ENABLE_CUDA)
```

Comment:
What is the reason behind defining both of these?

Reply:
Only the second is actually necessary

Reply:
Good point